NetKAN/KashCorpSA.netkan

```diff
@@ -0,0 +1,20 @@
+{
+        "spec_version" : "v1.4",
+        "identifier" : "KashCorpSA",
+        "name" : "KashCorp Shared Assets (Virtual mod)",
+        "abstract" : "Shared assets for KashCorp mods, based off KAP",
+        "comment" : "KashCorp Shared Assets based off KAP zip file",
+        "license" : "BSD-3-clause",
+        "version" : "1.0v",
+        "$kref" : "#/ckan/kerbalstuff/96",
+        "depends" : [
+                { "name"        : "ModuleManager" }
+        ],
+        "install" : [
+                {
+                        "find" : "KashCorp",
+                        "install_to" : "GameData",
+                        "filter" : "KAP"
```

Comment:
These are the shared ones.  The KAP speciofic ones are in the KAP netkan, which is in a different fork.  Im waiting until the shared one goes in before I put the PR in for this one, because I cant test my netkans against it until its there for DCKAN to find as a "depends".  I already have stand alone netkans that work individually, but they have a problem with the shared files.